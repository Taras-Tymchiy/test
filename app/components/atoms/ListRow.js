// @flow
import * as React from 'react';
import {
    Platform,
    StyleSheet,
    View
  } from 'react-native';
import StyledHOC from '../HOCs/StyledHOC';
import { SPACING_M, SPACING_S, COLOR_ELEMENT_BG, COLOR_ELEMENT_BORDER } from '../styles/constants';

const styles = StyleSheet.create({
  container: {
    backgroundColor: COLOR_ELEMENT_BG,
    borderBottomWidth: StyleSheet.hairlineWidth,
    borderColor: COLOR_ELEMENT_BORDER,
    padding: SPACING_M
  }
});

export default StyledHOC(View, styles.container);
