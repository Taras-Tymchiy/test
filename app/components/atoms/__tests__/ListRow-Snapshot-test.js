import React from 'react';
import renderer from 'react-test-renderer';
import ListRow from '../ListRow';
import { Text } from 'react-native';

it('renders a ListRow using Snapshots', () => {
  expect(renderer.create(
    <ListRow />
  )).toMatchSnapshot();
});

it('renders an ListRow with children using Snapshots', () => {
  expect(renderer.create(
    <ListRow>
      <Text>Test</Text>
    </ListRow>
  )).toMatchSnapshot();
});
