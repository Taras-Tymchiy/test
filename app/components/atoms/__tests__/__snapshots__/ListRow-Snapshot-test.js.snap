// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a ListRow using Snapshots 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#FFF",
        "borderBottomWidth": 0.5,
        "borderColor": "#CCC",
        "padding": 12,
      },
      undefined,
    ]
  }
/>
`;

exports[`renders an ListRow with children using Snapshots 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#FFF",
        "borderBottomWidth": 0.5,
        "borderColor": "#CCC",
        "padding": 12,
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Test
  </Text>
</View>
`;
